replicate/common/consts.go

```diff
@@ -12,4 +12,5 @@ const (
 	ReplicateToMatching             = "replicator.v1.mittwald.de/replicate-to-matching"
 	KeepOwnerReferences             = "replicator.v1.mittwald.de/keep-owner-references"
 	StripLabels                     = "replicator.v1.mittwald.de/strip-labels"
+	StripAnnotations                = "replicator.v1.mittwald.de/strip-annotations"
```

Comment:
I thought about replacing this double-negation with `KeepAnnotations: true`, but then it would conflict with `StripLabels`.